Makefile-test.am

```diff
@@ -126,7 +126,7 @@ test_unit_tcti_device_SOURCES = test/unit/tcti-device.c tcti/tcti_device.c log/l
 test_unit_tcti_socket_CFLAGS  = $(CMOCKA_CFLAGS) $(AM_CFLAGS) $(URIPARSER_CFLAGS)
 test_unit_tcti_socket_LDADD   = $(CMOCKA_LIBS) $(libmarshal) $(URIPARSER_LIBS) $(libutil)
 test_unit_tcti_socket_LDFLAGS = -Wl,--wrap=connect,--wrap=read,--wrap=select,--wrap=write
-test_unit_tcti_socket_SOURCES = tcti/platformcommand.c tcti/tcti_socket.c \
+test_unit_tcti_socket_SOURCES = tcti/tcti_platform_command.c tcti/tcti_socket.c \
```

Comment:
@tstruk : Any objections to putting the `tcti_platform_command` function directly in the `tcti_socket` module? It's the only function in that `c` file and the implementation requires access to the private portions of the socket TCTI context so it's very tightly coupled to the socket TCTI module.

Reply:
@flihp no problem I will do that after #843 will go in. This will need refactoring on top of that anyway.